exercises/clock/example_gen.go

```diff
@@ -15,7 +16,8 @@ func main() {
 		log.Fatal(err)
 	}
 	var j js
-	if err := gen.Gen("clock.json", &j, t); err != nil {
+	f := path.Join("exercises", "clock", "canonical-data.json")
```

Comment:
no worries, I won't repeat myself here, leave this PR for the time being and hopefully I'll have the `gen.Gen` function working this way soon.